### DISCUSSION TITLE
Duda formulario recorrección T0

### DISCUSSION THREAD

**Discussion by k0rval4n** [IIC2133-PUC/2024-2#79] [Tarea 0] [answered]
¡Hola! Qué se debe marcar en el formulario si obtengo un `RUNTIME ERROR` en el feedback de la T0?
Al ejecutar mi código en local no tengo errores al compilar, y tanto con los tests públicos como privados no saltan errores al ejecutar.
En el anuncio se indica que se debe marcar la opción `otro`, sin embargo, esta opción no existe como tal en el formulario.
![image](https://github.com/user-attachments/assets/2b13d950-8edd-401d-82f5-c981f895712a)

**Comment by sebacarrascop** [comment, marked as answer]
Hola!, acabamos de añadir la opción!

**Reply by k0rval4n** [reply]
Muchas gracias!
